### ISSUE TITLE
Ajaxify Your Dynamic Tags

### ISSUE THREAD

**Issue by NimaBakhtiyari** [CrocoBlock/suggestions#5952]
For Example If I Use To Query Count Dynamic Tag For Counting In Cart Items Results will Not Change Until Refreshing Page And Until This Feature Is Added To The Plugin, It Cannot Be Called A Real Dynamic Plugin And Many Ideas Will Remain Out Of Reach.

**Comment by azkalex** [comment]
I agree, I also have a scenario where I change the meta fields values on a page with a form that's inside a popup triggered by a button on that page. Since the forms can do AJAX, the dynamic tags should be able to read that and update themselves without having to add a redirection to the same page to trigger a refresh, or even manually refresh it.
